### PR TITLE
Replaced /boot/config.txt file with newer version

### PR DESCRIPTION
**Pull request by radujipa**
Numerous changes were made to the config.txt file in kano-settings
which we need align. Replacing the file rather than migrating each
change is more convenient here.

### CODE REVIEW

**Comment by radujipa** [comment]
[(8) [CARE] Screen Resolution issue when moving from TV to monitor. [8]](https://trello.com/c/vsFlFYaY/1172-8-care-screen-resolution-issue-when-moving-from-tv-to-monitor-8)

**Comment by Ealdwulf** [comment]
This means people will have to restore their audio/monitor settings. Might be a good idea to let Customer care know to expect that.

File: kano_updater/scenarios.py [hunk 1]
```diff
@@ -849,3 +849,13 @@ def _add_boot_config_options(self, extra_config):
     def beta_3_11_0_to_beta_3_12_0(self):
         # Remove .asoundrc files from all users (see kano-desktop & kano-settings).
         remove_user_files(['.asoundrc'])
+
+        # Replace the config.txt after numerous changes there (see kano-settings).
+        try:
+            import shutil
+            shutil.copyfile(
```
Comment (tombettany) [inline kano_updater/scenarios.py]: Can we at least backup their old file?

Comment (skarbat) [reply kano_updater/scenarios.py]: I encourage some sort of backup as well

**Comment by radujipa** [comment]
I've rebased this branch on the 3.13.0 image.

@tombettany @skarbat I've created a backup of the config file, good idea!
@Ealdwulf I've preserved the audio output channel after the config.txt replacing, thanks for spotting that!

**Comment by skarbat** [comment]
Seen it 👍

**Pull request merged**